### DISCUSSION TITLE
c++/cli template

### DISCUSSION THREAD

**Discussion by furesoft** [AvaloniaUI/Avalonia#9334] [Q&A] [answered]
Now c++/cli is supported on .net core, so could you make templates for it?

**Comment by kekekeks** [comment, marked as answer]
It's only supported for Windows. Other platforms aren't supported, so we currently don't see enough value in it to spend our limited resources on C++/CLI support.

PRs are always welcome.

**Comment by timunie** [comment]
Converted this into Q&A as this is neither a bug nor a feature request. If you disagree please let us know.
